gtk4/sys/src/lib.rs

```diff
@@ -11301,6 +11330,7 @@ extern "C" {
     pub fn gtk_file_filter_add_mime_type(filter: *mut GtkFileFilter, mime_type: *const c_char);
     pub fn gtk_file_filter_add_pattern(filter: *mut GtkFileFilter, pattern: *const c_char);
     pub fn gtk_file_filter_add_pixbuf_formats(filter: *mut GtkFileFilter);
+    pub fn gtk_file_filter_add_suffix(filter: *mut GtkFileFilter, suffix: *const c_char);
```

Comment:
This seems to lack the since v4_4 annotation

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3736